overlord/ifacestate/ifacemgr_test.go

```diff
@@ -377,8 +377,7 @@ slots:
 	s.mockIface(c, &interfaces.TestInterface{InterfaceName: "test"})
 
 	setup()
-
-	//mgr := s.manager(c)
+	_ = s.manager(c)
```

Comment:
As you just pointed out in telegram, there are a lot of uses ike this in this file without any comment, so based on this its fine. I do think we should look at this later and see if we can move s.manager(c) into some sort of setUp method to avoid this kind of duplication and magic (but not in this branch :)